.travis.yml

```diff
@@ -1,47 +1,72 @@
-
+sudo: false
+os:
+  - linux
+  - osx
 language: cpp
-
 compiler:
  - clang
 
 env:
- - GOPATH=${TRAVIS_BUILD_DIR}/go CPPFLAGS="-I/usr/src/gmock/include" CXXFLAGS="-Wall -Werror"
+  - CXXFLAGS="-std=c++11"
```

Comment:
Changed a bunch of stuff around this, I'm doing an explicit `matrix` now, which makes things easier in the `before_install` section.

Also, the `std=c++11` was a hangover from the gcc versions vs. clang versions vs. stdlib++ versions fight, and since I finally won that particular battle I've removed it.